### DISCUSSION TITLE
Expose `LocaleInputType` in exported GraphQL schema

### DISCUSSION THREAD

**Discussion by ghost** [payloadcms/payload#1014] [Feature Requests & Ideas]
Hi 👋🏻 

It could be cool to add `LocaleInputType` to the generated schema when you do `payload generate:graphQLSchema`, this way it would let us create strongly typed queries on front-end using GraphQL codegen.
It could be amazing to add locales as an enum within the schema too.

What do you think about it?

**Comment by ghost** [comment]
Erratum, `LocaleInputType` is already exported from `schema` made a mistake on my side 🤡

**Reply by ssyberg** [reply]
How would you reference this in a custom graphql endpoint definition?

**Reply by ssyberg** [reply]
Figured this out:
`locale: { type: payload.types.localeInputType },`
